Breathr-LLC/breathr
# Lr dev

##Overview
Issue Type

[] Bug
[x] Feature
[]Tech debt

Description
Added Journal components and updated webpack file to include React plugin.

Steps to Validate Feature
Navigate to /journal route

Previous Behavior
File did not exist.

Expected Behavior
Journal components will now appear in the app

How Has This Been Tested
See "Steps to Validate Feature"

Screenshots & Videos
N/A

Additional Context
N/A
